include/solution_transfer/meshfree_solution_transfer.h

```diff
@@ -21,6 +21,7 @@
 #define MESHFREESOLUTIONTRANSFER_H
```

Comment:
Although It's not part of your change, I just noticed that this include guard doesn't follow our usual pattern. So, these two lines could be changed to:
```
#ifndef MESHFREE_SOLUTION_TRANSFER_H
#define MESHFREE_SOLUTION_TRANSFER_H
```
if you felt so inclined.